### ISSUE TITLE
Insert #hashtag and @account in a single list view

### ISSUE THREAD

**Issue by nspt-fr** [tootsuite/mastodon#12367]
Enable List to accept #hashtag references and users @account at the same time.

It would be possible to implement it in third party apps using the REST API.

### Pitch

While searching on different topics, News, Art, Computer Science… We might want to combine #hashtag and @accounts inside a same list view.

### Motivation

This functionality will give a better approach of reading informations about specific topics. Might be usefull to everybody using List.

**Comment by ffffluffy** [comment]
I agree, and to be able to filter the users toots, like, if you want only the toots of `@user` containing the hashtag `#dinovember`.

**Comment by Thann** [comment]
Yes!  I came here to make this issue.  I think it would make it appeal more to #lemmy users.
